lumigator/python/mzai/sdk/lumigator_sdk/strict_schemas.py

```diff
@@ -0,0 +1,76 @@
+from lumigator_schemas.completions import CompletionResponse
+from lumigator_schemas.datasets import DatasetDownloadResponse, DatasetResponse
+from lumigator_schemas.experiments import (
+    ExperimentCreate,
+    ExperimentResponse,
+    ExperimentResultDownloadResponse,
+    ExperimentResultResponse,
+)
+from lumigator_schemas.extras import HealthResponse, ListingResponse
+from lumigator_schemas.jobs import (
+    JobConfig,
+    JobEvalCreate,
+    JobEvent,
+    JobInferenceCreate,
+    JobLogsResponse,
+    JobResponse,
+    JobResultDownloadResponse,
+    JobResultResponse,
+    JobSubmissionResponse,
+)
+from pydantic import ConfigDict
+
+
+class CompletionResponse(CompletionResponse):
```

Comment:
Looks good, thanks for this! Minor question: what happens when CompletionResponse is updated in schemas and we use it as a dependency here? Do we need to have any checks/testing in place?

Reply:
I guess that if the schemas version changes, and the test jsons are no longer compliant with the schemas, we'll seee the SDK tests failing. That'd be the indication that we need to bump the schemas version in the SDK. But the implementation would be more or less the same, since it just takes the base class and adds some config on top of it.

Reply:
ok so what I'm hearing is we pin at the PIP level for now, makes sense

Reply:
Keeping unresolved in case you want to discuss this further, otherwise please resolve this item.

Reply:
Resolving this issue. Dependencies in pyproject.toml get included in PKG-INFO. The editable annotation seems to be useful only for uv when running from sources directly. But this means that we may want to add some additional check across sdk/schemas versions as part of the release process.